test/Scripts.Integration.Test/integration-test.ps1

```diff
@@ -1,7 +1,7 @@
 ﻿param($path)
 
-.\test/Scripts.Integration.Test/integration-create-project.ps1 "$path"
-.\test/Scripts.Integration.Test/integration-build-project.ps1 "$path"
-.\test/Scripts.Integration.Test/integration-update-sentry.ps1 "$path"
-.\test/Scripts.Integration.Test/integration-build-project.ps1 "$path"
-.\test/Scripts.Integration.Test/integration-run-smoke-test.ps1
+.\test/Scripts.Integration.Test/create-project.ps1 "$path"
+.\test/Scripts.Integration.Test/build-project.ps1 "$path"
+.\test/Scripts.Integration.Test/update-sentry.ps1 "$path"
+.\test/Scripts.Integration.Test/build-project.ps1 "$path"
+.\test/Scripts.Integration.Test/run-smoke-test.ps1
```

Comment:
BTW this file is (likely) broken - it hasn't been updated with all the changes for the last few months. Do we want to keep & fix it?